src/lib/card/card.scss

```diff
@@ -16,6 +16,20 @@ $mat-card-header-size: 40px !default;
   padding: $mat-card-default-padding;
   border-radius: $mat-card-border-radius;
 
+  .mat-divider {
```

Comment:
can you separate the attributes from the rules here?

```scss
.mat-divider {
  position: absolute;
  left: 0;
  width: 100%;

  [dir='rtl'] & {
    left: auto;
    right: 0;
  }

  &.mat-divider-inset {
    position: static;
    margin: 0;
  }
}
```

Reply:
Fixed